### PR TITLE
fix/slow down testcafe tests

### PR DESCRIPTION
**Pull request by osslgtm**
## Summary

Fixing CI frequent Timeouts and Test Failures

## Changes

Slowing down of testcafe tests by using the --speed flag
Caveats: Slower CI Process for tests

## Issues

All the failing PRs

## Details

https://testcafe.io/documentation/402963/guides/advanced-guides/speed-up-test-execution#set-test-speed
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions

### CODE REVIEW

**Comment by simboonlong** [comment]
might burn minutes but i can understand the false positive woes, was thinking if:
- identify the usual failing tests, litter them with timeouts <- not really keen on this tho
- identify the usual failing tests, refactor them to simpler, smaller tests without losing tests' intent

not sure what the rest think

**Comment by osslgtm** [comment]
Aren't the minutes free for public repos ?
https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions

Maybe we can switch it to running for PR only ?
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Time-wise i don't think the gap is that big even though the speed is set to half
https://github.com/TradeTrust/tradetrust-website/actions?query=is%3Asuccess

I'll fine grain the speed and identify the slower running tests during the later commits, but i think a blanket speed will help for now

**Comment by simboonlong** [comment]
ok, can get this in and monitor later

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "integration": "testcafe chrome src/**/*.spec.ts",
     "integration:single": "testcafe chrome -L",
     "integration:headless": "testcafe -c 4 chrome:headless src/**/*.spec.ts --app \"npm run serve-static\"",
+    "integration:headless:slow": "testcafe -c 4 chrome:headless src/**/*.spec.ts --speed 0.5 --app \"npm run serve-static\"",
```
Comment (simboonlong) [inline package.json]:
```suggestion
    "integration:headless": "testcafe -c 4 chrome:headless src/**/*.spec.ts --speed 0.5 --app \"npm run serve-static\"",
```

**Review by simboonlong — APPROVED** [review]

**Comment by osslgtm** [comment]
sorry but i think the fix is not good enough, i've been trying to replicate the issue and it's frequently stuck in between validateTextContent and validateIframeTexts.

i think concurrency is part of the problem as it's a 2 core vm having to support 4 chrome windows but i think i need to figure a way to resolve the root issue

**Comment by osslgtm** [comment]
Better fix on #558

**Pull request closed without merging**